token/program-2022/tests/program_test.rs

```diff
@@ -52,14 +69,19 @@ impl TestContext {
         let mint_account = Keypair::new();
         let mint_authority = Keypair::new();
         let mint_authority_pubkey = mint_authority.pubkey();
+        let freeze_authority = if enable_freeze {
+            Some(&mint_authority_pubkey)
+        } else {
+            None
+        };
```

Comment:
nittiest of nits, but how about a separate `Option<Keypair>` field on `TokenContext` for this?  Then we can be sure that we aren't reusing the same authority elsewhere by accident.